projects/portal/src/app/views/staking/create-validator-simple/create-validator-simple.component.html

```diff
@@ -0,0 +1,255 @@
+<h2>Upload file here to be Validator!</h2>
+<form class="h-screen" #formRef="ngForm">
+  <input
+    #fileInputRef
+    name="fileInput"
+    type="file"
+    multiple="false"
+    accept="image/*"
+    (change)="onChangeFile($event)"
+  />
+  <!-- <mat-accordion>
```

Comment:
Do you need to erase this place off?

Reply:
@YasunoriMATSUOKA 
olz check it

Reply:
@mkXultra 
Thank you.
I left them for debug purpose. (When something goes wrong, developer will need to check the hidden form values.)
But, I think your point is correct.
I think it might be a good idea to leave the above intent as a comment.
How do you think about it?

Reply:
@YasunoriMATSUOKA 
Uncomment and commit once.
Then comment out and commit.

It will remain in the commit log and you can restore it at any time.

Reply:
@mkXultra 
Thank you.
I got it.
Wait a moment plz.

Reply:
@mkXultra 
I fixed it. Plz review again 🙏